### PR TITLE
Fix Collection.get fallback

### PR DESCRIPTION
**Pull request by lancetipton**
### Updates
  * Fixed get not returning fallback in some cases
    * Added extra test case
  * Added no-cache to jest tests

### CODE REVIEW

File: src/scripts/__tests__/collection.js [hunk 1]
```diff
@@ -40,7 +40,14 @@ describe('/collection', () => {
...
-      expect(Coll.get(getObj, path, "fallback") === 'fallback').toBe(true)
+      expect(Coll.get(getObj, path, "deep_fallback") === 'deep_fallback').toBe(true)
+    })
+
+    it('should return a fallback when a value in the path does not exist', () => {
+      const getObj = { data: [ { exists: 'I Exist!' } ] }
+      const path = 'data.1.test'
+
+      expect(Coll.get(getObj, path, "shallow_fallback") === 'shallow_fallback').toBe(true)
     })
```
Comment (mpcarolin) [inline src/scripts/__tests__/collection.js]: I would also test that it returns the fallback when the object itself is null, but I already know that works.
